app/javascript/components/modals/TaskHintsModal.tsx

```diff
@@ -14,20 +14,30 @@ export const TaskHintsModal = ({
   onClose: () => void
 }): JSX.Element => {
   return (
-    <Modal open={open} onClose={onClose} className="m-editor-hints" {...props}>
-      <header>
-        <GraphicalIcon icon="hints" category="graphics" />
-        <h2>Hints and Tips</h2>
-      </header>
-      <div className="single-task-hints">
-        <h3>{task.title}</h3>
-        {task.hints.map((hint, idx) => (
-          <div
-            className="c-textual-content --large"
-            key={idx}
-            dangerouslySetInnerHTML={{ __html: hint }}
-          ></div>
-        ))}
+    <Modal
```

Comment:
Assuming this sets the correct role and aria properties...

Reply:
If this does not add a hidden close button you may want to add one yourself but that can be a separate PR.